pkgs/tools/typesetting/satysfi/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchzip, fetchFromGitHub, ocaml, ocamlbuild, findlib, menhir,
+  ppx_deriving, uutf, result, core_kernel, bitv, batteries, yojson, camlimages }:
+let
+  lm = fetchzip {
+    url = "http://www.gust.org.pl/projects/e-foundry/latin-modern/download/lm2.004otf.zip";
+    sha256 = "1mc88fbhfd2wki2vr700pgv96smya6d1z783xs3mfy138yb6ga2p";
+    stripRoot = false;
+  };
+  lm-math = fetchzip {
+    url = "http://www.gust.org.pl/projects/e-foundry/lm-math/download/latinmodern-math-1959.zip";
+    sha256 = "15l3lxjciyjmbh0q6jjvzz16ibk4ij79in9fs47qhrfr2wrddpvs";
+    stripRoot = false;
+  };
+in
+  stdenv.mkDerivation rec {
+    name = "satysfi-${version}";
+    version = "2018-03-07";
+    src = fetchFromGitHub {
+      owner = "gfngfn";
+      repo = "SATySFi";
+      rev = "a050ec0906d083682c630b0dea68887415b5f53d";
+      sha256 = "12bhl7s2kc02amr8rm71pihj203f2j15y5j0kz3swgsw0gqh81gv";
+      fetchSubmodules = true;
+    };
+    buildInputs = [ findlib menhir ppx_deriving uutf result core_kernel bitv
+      batteries yojson camlimages ];
+    nativeBuildInputs = [ ocaml ocamlbuild ];
```

Comment:
Why this split between `buildInputs` and `nativeBuildInputs`?

Reply:
I guessed ocaml and ocamlbuild were only needed at build time. So this should all be in buildInputs?